docs/cli-commands.md

```diff
@@ -9,18 +9,39 @@ Most of the commands documented on this page
 assume that you know how to address the subdomains of a DSP server.
 There are three relevant URLs you should know about:
 
-- Subdomain `admin` stands for the DSP-APP frontend that you look at in your browser
+- Subdomains `admin`/`app` stand for the DSP-APP frontend that you look at in your browser
 - Subdomain `api` stands for the DSP-API (where DSP-TOOLS sends its data to) 
 - Subdomain `iiif` stands for the SIPI server interface (where DSP-TOOLS sends the multimedia files to)
 
-This means that for uploading data to a DSP server 
+This means that for uploading data to the DSP server 
 on the domain `dasch.swiss`, 
 you have to type the following:
 
 ```bash
-dsp-tools xmlupload -s https://api.dasch.swiss -u root@example.com -p 'test' -S https://iiif.dasch.swiss xml_data_file.xml
+dsp-tools xmlupload -s https://api.dasch.swiss -u 'your@email.com' -p 'password' xml_data_file.xml
```

Comment:
If I reformulate your suggestion: You suggest adding the feature to resolve `-s prod|test|staging|dev` to the according servers. Because the other 3 points are already implemented.

I think this is a nice to have, but would introduce some overhead

Reply:
True, if all the other things already work, then my suggestion was only that. I didn't have the full CLI in my head when I wrote my comment.

And if it's only that, then it may be nice, but it's probably not too pressing, so if it causes overhead (though I feel this should be pretty simple) maybe it's not worth doing